backend/game/cards/discard_cards/victory_point_manipulation_cards/card_collector.py

```diff
@@ -0,0 +1,36 @@
+from os.path import dirname, basename, isfile, join
+import glob
+from typing import List
+
+from game.cards.card import Card
+
+
+def get_class(class_string_name_and_path):
+    parsed_class_string = class_string_name_and_path.split('.')
+    module = ".".join(parsed_class_string[:-1])
+    m = __import__(module)
+    for comp in parsed_class_string[1:]:
+        m = getattr(m, comp)
+    return m
+
+
+def get_list_vpm_cards(prepend_path):
```

Comment:
Initial gut reaction is I do not like this because I don't think it's clean or substainable, I will need to look more into a better example of how to go about this but I think we can utilitize the __init__ files for this purpose.  Yes we might have to list out all the cards but I think that's okay as it'll only need to be done once

https://www.bensnider.com/dynamically-import-and-instantiate-python-classes.html
http://www.pressthered.com/dynamically_importing_and_instantiating_modules_in_python/

Reply:
Yes after playing around with the `__init__` files a bit, you can probably get them to properly call the subclasses here so you do not have to dynamically find them.  In the end, when you import cards, basically it makes sense for all cards to be imported.  I think that may be better to try before coming up with this sort of dynamic function

Reply:
I agree. Also there was multiples of some cards, so I'd like the deck to be listed out manually as well.